doctrine/dbal.rst

```diff
@@ -20,6 +20,8 @@ makes it easy to execute queries and perform other database actions.
 
     Read the official Doctrine `DBAL Documentation`_ to learn all the details
     and capabilities of Doctrine's DBAL library.
+    
+    Before you start you will need `DoctrineBundle`_ which supports both Doctrine ORM and Doctrine DBAL (together or just DBAL). This will generate ``config/packages/doctrine.yaml``.
```

Comment:
According to how we do that in other articles I would write something like this:

```rst
First, install the Doctrine bundle:

.. code-block:: terminal

    composer require doctrine/doctrine-bundle
```